File: 6-using-regular-expressions/1.rb
def url?(text)
  p !!text.match(/\Ahttps?:\/\/\S+\z/)
end

url?('http://launchschool.com')   # -> true
url?('https://example.com')       # -> true
url?('https://example.com hello') # -> false
url?('   https://example.com')    # -> false

File: 6-using-regular-expressions/1.js
const isUrl = function(text) {
  console.log(!!text.match(/^https?:\/\/\S+$/));
  return !!text.match(/^https?:\/\/\S+$/);
}

isUrl('http://launchschool.com');   // -> true
isUrl('https://example.com');       // -> true
isUrl('https://example.com hello'); // -> false
isUrl('   https://example.com');    // -> false
